_redirects

```diff
@@ -162,7 +162,7 @@
 /planungen/9070/* /planungen/223/:splat 301
 
 /stats/* https://s.fixmycity.de/:splat 200
+/info https://www.fixmycity.de/wer-wir-sind
```

Comment:
Doku für mich: `301` ist default https://docs.netlify.com/routing/redirects/#syntax-for-the-netlify-configuration-file